src/ZoneServer/Skills/Handlers/Archer/ObliqueShot.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Melia.Shared.L10N;
+using Melia.Shared.Tos.Const;
+using Melia.Zone.Network;
+using Melia.Zone.Skills.Combat;
+using Melia.Zone.Skills.Handlers.Base;
+using Melia.Zone.Skills.SplashAreas;
+using Melia.Zone.World.Actors;
+using Melia.Zone.World.Actors.CombatEntities.Components;
+using Yggdrasil.Util;
+using static Melia.Zone.Skills.SkillUseFunctions;
+
+namespace Melia.Zone.Skills.Handlers.Archer
+{
+	/// <summary>
+	/// Handles the Archer skill Oblique Shot.
+	/// </summary>
+	[SkillHandler(SkillId.Archer_ObliqueShot)]
+	public class ObliqueFire : ITargetSkillHandler
+	{
+		/// <summary>
+		/// Handles the skill, shoot missile at enemy that spreads to another target.
+		/// </summary>
+		public void Handle(Skill skill, ICombatEntity caster, ICombatEntity designatedTarget)
+		{
+			if (!caster.TrySpendSp(skill))
+			{
+				caster.ServerMessage(Localization.Get("Not enough SP."));
+				return;
+			}
+
+			skill.IncreaseOverheat();
+			caster.TurnTowards(designatedTarget);
+
+			if (designatedTarget == null)
+			{
+				return;
+			}
+
+			if (!caster.Position.InRange2D(designatedTarget.Position, skill.Data.MaxRange))
+			{
+				caster.ServerMessage(Localization.Get("Too far away."));
+				return;
+			}
+
+			this.Attack(skill, caster, designatedTarget);
+		}
+
+		/// <summary>
+		/// Executes the actual attack.
+		/// </summary>
+		/// <param name="skill"></param>
+		/// <param name="caster"></param>
+		/// <param name="designatedTarget"></param>
+		private async void Attack(Skill skill, ICombatEntity caster, ICombatEntity designatedTarget)
+		{
+			var damageDelay = TimeSpan.Zero;
+			var skillHitDelay = TimeSpan.Zero;
+			var secondHitTime = TimeSpan.FromMilliseconds(100);
+
+			var skillHitResult = SCR_SkillHit(caster, designatedTarget, skill);
+			designatedTarget.TakeDamage(skillHitResult.Damage / 2, caster);
```

Comment:
Is splitting the damage between main target and sub-target official behavior?

Reply:
I probably miss understood, I remember seeing a comment about it somehwer. But its wrong.
The damage should split at all.